draft-tuexen-tsvwg-dtls-chunk-key-management.xml

```diff
@@ -110,15 +110,62 @@ they appear in all capitals, as shown here.</t>
 
 <section>
 <name>Architecture</name>
-<t>TBD</t>
+<t>This document describes how Datagram Transport Layer Security (DTLS) 1.3 is
+used to establish keys for securing SCTP using the DTLS Chunk. This approach
+combines the performance and encryption flexibility of DTLS Chunks with the
+integrated key management capabilities of DTLS. The resulting solution
+can be viewed from two perspectives: as a new key management method for
+SCTP, and as an SCTP-specific encryption layer integrated with DTLS.
+</t>
+
+<t>The key characteristics of the solution are as follows:</t>
+<ul>
+<li><t>Application data is protected using DTLS Chunks.</t></li>
+<li><t>The DTLS handshake is used to establish keying material, algorithms,
+and parameters for use with DTLS Chunks.</t></li>
+<li><t>DTLS and relevant extensions are used to negotiate cryptographic
+algorithms and parameters, perform extended key updates, enable larger
+record sizes, and support post-handshake authentication.</t>
+<li><t>All other DTLS record-layer content types are protected using
+standard DTLS record framing.</t></li>
+</ul>
+
+<t>Application data is never transmitted in DTLS record-layer application_data
+records. Instead, application data is passed to the DTLS stack, which forwards
+it to the Record Protection Operator. The operator encapsulates the data into
+DTLS Chunks, applying appropriate protection.</t></li>
```

Comment:
I don't think this is true. application data it put into SCTP data chunks and then protected by the DTLS chunk protector, not the DTLS record protector.